client/my-sites/plan-features/index.jsx

```diff
@@ -288,6 +296,55 @@ class PlanFeatures extends Component {
 		} );
 	}
 
+	renderFeaturePopover() {
+		return (
+			<Popover
+				showDelay={ 100 }
+				id="popover__plan-features"
+				isVisible={ this.state.showPopover }
+				context={ this.state.popoverReference }
+				position="right"
+				onClose={ this.closeFeaturePopover }
+				className={ classNames(
+						'info-popover__tooltip',
+						'popover__plan-features'
+					) }
+				>
+					{ this.state.popoverDescription }
+			</Popover>
+		);
+	}
+
+	renderFeatureItem( feature, index ) {
+		return (
+			<PlanFeaturesItem
+				key={ index }
+				description={ feature.getDescription
+					? feature.getDescription()
+					: null
+				}
+				onMouseEnter={ this.showFeaturePopover }
+				onMouseLeave={ this.closeFeaturePopover }
+			>
+				{ feature.getTitle() }
+			</PlanFeaturesItem>
+		);
+	}
+
+	showFeaturePopover( el, popoverDescription ) {
+		this.setState( {
+			showPopover: true,
+			popoverDescription,
+			popoverReference: el
+		} );
+	}
+
+	closeFeaturePopover() {
+		this.setState( {
+			showPopover: false,
```

Comment:
We could also set the `popoverReference` to `null` here as well as `popoverDescription: ''` to clear the state and make it the same as default state.